code/src/layouts/header/components/header-search-input-dropdown/HeaderSearchInputDropdown.tsx

```diff
@@ -0,0 +1,114 @@
+import AppBox from "@/components/app-box/AppBox";
+import AppLink from "@/components/app-link/AppLink";
+import AppMenuItem from "@/components/app-menu-item/AppMenuItem";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+import noResultsImage from "@/assets/images/search/no-results.png";
+import routePaths from "@/constants/routes";
+import { ProductFromSearch } from "@/types/product.types";
+
+import "@/layouts/header/components/header-search-input-dropdown/HeaderSearchInputDropdown.scss";
+
+type HeaderSearchInputDropdownProps = {
+  isError: boolean;
+  searchResults: ProductFromSearch[];
+  handleCloseDropdown: () => void;
+  totalElements: number;
+  isLoading: boolean;
+};
+
+const HeaderSearchInputDropdown = ({
+  handleCloseDropdown,
+  searchResults,
+  totalElements,
+  isError,
+  isLoading
+}: HeaderSearchInputDropdownProps) => {
+  const errorLabel = (
+    <AppTypography component="li" translationKey="errors.somethingWentWrong" /> // will be replaced by error layout
+  );
```

Comment:
Mark as `@TODO` to make global search easier

Reply:
Marked as @TODO